public/build/CNAME

```diff
@@ -1 +0,0 @@
-mobymask.com
```

Comment:
Since mobymask is shipped by GitHub Pages, it is important to verify the build workflow and confirm that this file exists in the root folder of the `gh-pages` branch.

Reply:
Also, vercel is another good choice with professional CDN service.

Reply:
Should be fine.